### ISSUE TITLE
Password Obfuscation

### ISSUE THREAD

**Issue by Ajt8000** [FossifyOrg/Keyboard#63]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Keyboard/issues) or [closed](https://github.com/FossifyOrg/Keyboard/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Keyboard/discussions) or [closed](https://github.com/FossifyOrg/Keyboard/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Keyboard/blob/main/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

When copying passwords from a password manager, the password appears in the keyboard's top bar unobscured.
It shows the password as text rather than as a series of dots or asterisks.
This is a security issue as if someone was looking at my phone screen at the time they would now know my password for the app I was logging into.

### Why do you want this feature?

For security sake.

**Comment by Aga-C** [comment]
Does any keyboard and password manager provide such a function? 

I've tested KeePassDX and Firefox's password manager with two proprietary keyboards: Gboard and SwiftKey. For both password managers and both keyboards, I've always seen a copied password written in plain text.

**Comment by Ajt8000** [comment]
I've had this work by default with AnySoftKeyboard combined with Bitwarden as the password manager.

I'm using Bitwarden with the Fossify keyboard as well.

**Comment by Aga-C** [comment]
I've just installed Bitwarden, and in AnySoftKeyboard I see the copied password as a text, not encrypted in any way.

![signal-2024-07-29-11-46-55-601.jpg](https://github.com/user-attachments/assets/7cc97d0d-d9dd-4df4-bd1b-ca9a94bdb5cf)

**Comment by Ajt8000** [comment]
It worked for me here. Regardless, it's a good feature to have, and really should be considered for Fossify.

![Screenshot_20240729-121516_Trebuchet](https://github.com/user-attachments/assets/986d4d95-d1f5-493d-bc14-78c589030969)

**Comment by Aga-C** [comment]
I did some investigation and found out, that Gboard and AnySoftKeyboard display clipboard contents as asterisks, however it's not dependent on what the clipboard has, but on the type of the text field. I can copy any text, and after selecting the password field, the keyboard doesn't show clipboard contents as plain text.

**Comment by Ajt8000** [comment]
That makes a lot of sense!
Would it be possible to add this feature to Fossify?

**Comment by naveensingh** [comment]
Yes, but as Aga-C described above, it'll depend on the type of the text field in focus. The password will still be visible as plain text if you are, for example, typing into a browser.

It's hard to figure out when to redact something without tricks like monitoring your app usage and your clipboard 24x7 and that is bad.

**Comment by sosasees** [comment]
there could be a settings&nbsp;toggle for always obscuring the clipboard&nbsp;content by default, with an 👁️&#x2060;eye&nbsp;button to show it
